osidb/tests/test_flaw.py

```diff
@@ -866,3 +868,13 @@ def test_validate_major_incident_summary(
                 flaw1.save()
         else:
             assert flaw1.save() is None
+
+    @freeze_time(tzdatetime(2021, 11, 23))
+    def test_validate_embargoing_public_flaw(self):
+        flaw = FlawFactory(embargoed=False)
+        with pytest.raises(ValidationError, match="Embargoing a public flaw is futile"):
```

Comment:
yep, and it accepts regex too, I was looking for something like this because the pattern we currently use (`assert str(e) == "foo"`) is not that solid

Reply:
it's easier to debug too because pytest will tell you that the `ValidationError` was raised, but the error message did not match

Reply:
Cool! Yes I agree that this is a lot better. We might refactor all the occurrences of the old exception message check to this one if we have some time.